### PR TITLE
Add aarch64 opensuse tumbleweed

### PR DESCRIPTION
**Pull request by replicajune**
This merge request is related to the issue #1011 and is aiming at being able to user OpenSuse with the current script approach to package manager detection

### CODE REVIEW

File: pkg/cidata/cidata.TEMPLATE.d/boot/30-install-packages.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ if [ "${LIMA_CIDATA_UDP_DNS_LOCAL_PORT}" -ne 0 ] || [ "${LIMA_CIDATA_TCP_DNS_LOC
 fi
 
 # Install minimum dependencies
-if command -v apt-get >/dev/null 2>&1; then
+if hexdump -C -n 4 "$(command -v apt-get)" | grep -qF 'ELF' >/dev/null 2>&1; then
```
Comment (AkihiroSuda) [inline pkg/cidata/cidata.TEMPLATE.d/boot/30-install-packages.sh:26]: Could you add a code comment to explain the background?

**Comment by AkihiroSuda** [comment]
Thanks, but please sign the commit for DCO
https://github.com/apps/dco

(run `git commit -a -s --amend`, and make sure that the `Signed-off-by: NAME <EMAIL>` line with your real name is included in the commit message)

**Comment by replicajune** [comment]
hello @AkihiroSuda, thank you for the time you're spending on the PR, much appreciated :)

- I've updated the two initial commits I made and signed them. It passes DCO now. Does it work for you ?
- I've added a comment explaining the chained command in the script, I've also added a detail about why it's done and a link to this PR in case it can be reverted in the future

**Comment by AkihiroSuda** [comment]
Thanks, but please use real name

**Comment by replicajune** [comment]
the name I use on github (and for other tech related stuff) comply with the definition of real name of DCO as far as I can see (https://github.com/cncf/foundation/blob/main/dco-guidelines.md: "Your real name is the name you convey to people in the community for them to use to identify you as you")

is this ok for you ?

**Comment by AkihiroSuda** [comment]
> the name I use on github (and for other tech related stuff) comply with the definition of real name of DCO as far as I can see (https://github.com/cncf/foundation/blob/main/dco-guidelines.md: "Your real name is the name you convey to people in the community for them to use to identify you as you")

Thanks, wasn't aware of that guideline.


> is this ok for you ?

Maybe.
Opened https://github.com/cncf/toc/issues/906 for confirmation about "pseudonyms"

**Comment by AkihiroSuda** [comment]
Also opened:
- https://github.com/cncf/foundation/issues/407

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**